general/administration/installing.md

```diff
@@ -27,25 +32,59 @@ The Jellyfin Docker image is available on [Docker Hub](https://hub.docker.com/r/
     `--volume /path/to/cache:/cache \`  
     `--volume /path/to/media:/media \`  
     `--net=host \`  
-    `jellyfin/jellyfin`  
+    `jellyfin/jellyfin:latest`  
   
 Alternatively, using docker-compose:  
 ```
 version: "3"  
 services:  
-    jellyfin:  
-      image: jellyfin/jellyfin  
-      network_mode: "host"  
-      volumes:  
-        - /path/to/config:/config  
-        - /path/to/cache:/cache  
-        - /path/to/media:/media  
+  jellyfin:  
```

Comment:
None of the changes below are necessary or have no effect. The examples provided in the documentation should be as minimal as possible.

Please remove the below changes.

Reply:
Remove Remove the changes to this section.

Reply:
It fixed the indentation to be 2 spaces vice 4 spaces. I also added the restart policy since it was missing. Restarting docker would cause your container to not start due to the default restart policy being 'off'. 

If I'm missing something important, please give more detail to the issue at hand.

Reply:
This example should be as minimal as possible. Providing opinions like `name` and `restart` makes it more difficult to pick out the important parts.

Also - restarting is not always ideal ex. permission issue.